### DISCUSSION TITLE
Problem implementing second deck with brainbrew

### DISCUSSION THREAD

**Discussion by boydkelly** [ohare93/brain-brew#38] [General]
Hi there,

I'm really not at all sure where this problem is coming from and so I didn't want to log an 'issue' at this point.  I have created an issue with crowdanki where I click on cancel after a change note type dialog, and despite this the import proceeds.   However its really the underlying reason I click cancel that is more important.  

(see https://github.com/Stvad/CrowdAnki/issues/143)

Just wondering do you have any initial thoughts on where this might be coming from?   

Thanks very much!!!!

**Comment by jeprecated** [comment]
Yes we've heard of this cancel issue before :cry: I was under the impression it was fixed, but I suppose not released? I will reply on the original issue :+1: if there are any Brain Brew issues then we can open them, no problem :sweat_smile:

**Reply by boydkelly** [reply]
Thanks for commenting over there.  I got it sorted out with the script provided.   Actually I don't at this point in time even push my crowdanki /brainbrew deck.json to github.   Since I don't share it, but do a ton of manipulation locally, I tried it and it just creates another step for me.   My brainbrew tsv is really the source of truth and I do keep this and yaml configs under version control.   Just an fyi, I have all this going with brainbrew:

1) My anki deck with 2000+ notes
2) [Dictionary](https://coastsystems.net/fr/lexique-jula-2021/) (yaml with xsl transform to update brainbrew tsv and push to anki and publish also via asciidoctor to html)  
3) Some revealjs slide decks:   [Jula vocab ](https://slides-dyu.coastsystems.net/index.html) 
[Money](https://slides-dyu.coastsystems.net/money.html)
 [Colors](https://slides-dyu.coastsystems.net/colors.html)

Thank you!!!
